core/src/main/java/org/kiwix/kiwixmobile/core/extensions/FragmentExtensions.kt

```diff
@@ -39,7 +39,7 @@ fun Fragment.toast(stringId: Int, length: Int = Toast.LENGTH_LONG) {
 fun Fragment.closeKeyboard() {
   val inputMethodManager =
     requireContext().getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
-  inputMethodManager.toggleSoftInput(InputMethodManager.HIDE_IMPLICIT_ONLY, 0)
```

Comment:
This is here for the toggle input keyboard. We shouldn't change that.

Reply:
sorry for that, I removed that change.